Sources/PenguinStructures/AnyArrayBuffer.swift

```diff
@@ -72,9 +52,9 @@ public struct AnyArrayBuffer<Dispatch: AnyObject> {
   
   /// A bounded contiguous buffer comprising all of `self`'s storage.
   public var storage: Storage?
-  public let dispatch: Dispatch.Type
+  public let dispatch: Dispatch
```

Comment:
A doc comment for this field would be very helpful.

Reply:
```suggestion
  /// A “vtable” of functions implementing type-erased
  /// operations that depend on the Element type.
  public let dispatch: Dispatch
```